metadata/nz_building_outlines.xml

```diff
@@ -312,7 +306,36 @@ If you adapt this work in any way or include it in a collection, and publish, di
+            <gco:CharacterString>
+**Source imagery**
+Resolution of the aerial imagery used for building outline capture ranges from 50cm pixel resolution (0.5m GSD) to 7.5cm pixel resolution (0.075m GSD).  The source imagery is linked via the external_source_id attribute of building outlines to the imagery_survey_id of NZ Imagery Surveys. Using this link, additional attributes can be connected to building outlines - for example the dates that the imagery was captured and the accuracy / ground sample distance of the imagery used.
+
+The NZ Imagery Surveys data dictionary is here: https://nz-imagery-surveys.readthedocs.io/en/latest/index.html
+
+**Accuracy Specification:**
+The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source aerial imagery. The building outlines were created by an both an automated and manual method of feature extraction. Outlines are captured in full where they are partially occluded by vegetation or require additional viewer interpretation, and squared off at 90 degrees if the corner angles are between 80-100 degrees.
+
+**The following criteria were used for capturing buildings as building outlines:**
+• Buildings under construction, caravans, caravan trailers, house boats, shipping containers and other mobile structures are not captured as building outlines.
+• Primary building structures are captured as separate building outline polygons when the roofs are separate from nearby building roof structures.
+• Building extensions, sunrooms, balconies, patios and annexes are captured as part of the primary building outline structure if they exist as part of multi-level buildings and if sufficient imagery resolution allows this determination.
+• Permanent building structures such as sheds and greenhouses >10sq meters, not attached to a primary building structure, are captured as a separate building outlines.
+• Adjoining townhouses are not captured as separate structures, but rather as joined primary structures.
+• Adjoining commercial buildings are captured as separate building outlines when rooflines allows delineation.
+• Hay bales and silage are not captured as building outlines.
+• Building outline polygons captured will be greater than 10 square meters. An exception to this is in national parks, conservation areas, and wildlife areas where small structures such as huts, bivies and shelters are included as building outlines to address safety concerns in remote locations
+• Water tanks are captured as building outlines when their area is at least 16.5 square metres (or 4.5 meters in diameter).      
+
+**Valid Geometry:**    
+A building outline polygon is considered to have valid geometry if;
```

Comment:
Should we list all the validation check we've done? if so, might need to add 'It does not contain duplicate nodes'.
And what about the 'not outside the coastline' and 'not intersect with water features'?

Reply:
Our not-outside-coastline and intersect-water-feature are too complicated to explain since there are some valid buildings which violate those conditions.